cmd/release-controller/types.go

```diff
@@ -220,6 +224,8 @@ type ReleaseVerification struct {
 	ProwJob *ProwJobVerification `json:"prowJob"`
 	// Maximum retry attempts for the job. Defaults to 0 - do not retry on fail
 	MaxRetries int `json:"maxRetries,omitempty"`
+	// AnalysisJobCount Number of asynchronous jobs to execute for release analysis. Defaults to 10
```

Comment:
You don't want this to default.

Reply:
I have removed the default, and the maximum, for the AnalysisJobCount.